Answers.md

```diff
@@ -1 +1,20 @@
+# Sprint Challenge - Answers
 
+**1. Describe the biggest difference between .forEach & .map.**
+.forEach iterates through an array without returning anything and .map creates a new array and returns the results.
+
+
+**2. What is the difference between a function and a method?**
+A method is a function that is defined as a property to something else.
+
+
+**3. What is closure?**
+Closure relates to the scope of a function within a function and the variables it does and doesn't have access to.
+
+
+**4. Describe the four rules of the 'this' keyword.**
+Window, implicit, explicit, and new.
```

Comment:
The question was meant to give you an opportunity to show that you understand the meaning of the four rules. Make sure that you take a second look at these. These rules will continue to be used throughout the program, all the way through react.